tests/PageObjects/SongLanguagesPage.ts

```diff
@@ -23,10 +27,29 @@ export class SongLanguagesPagePO {
   }
 
   public async ensureSongLanguageIsSelected(language: string) {
-    const attributeValue = await this.getCheckbox(language).getAttribute('data-testid');
+    if (!(await this.isLanguageSelectedStr(language))) {
+      await this.getCheckbox(language).click();
+    }
+  }
+
+  private async isLanguageSelectedStr(language: string) {
+    const languageCheckbox = this.getCheckbox(language);
+    return this.isLanguageSelected(languageCheckbox);
+  }
+
+  private async isLanguageSelected(language: any) {
+    const languageCheckboxValue = await language.getAttribute('data-testid');
+    return languageCheckboxValue === 'CheckBoxIcon';
+  }
+
+  public async ensureAllLanguagesAreSelected() {
+    await expect(this.page.locator('[data-test^="lang-"] svg').first()).toBeVisible();
```

Comment:
Move `this.page.locator('[data-test^="lang-"] svg')` (or just `'[data-test^="lang-"] svg'` if it doesn't work)  to a new local variable